core-web/src/main/java/org/silverpeas/core/webapi/calendar/AbstractEventEntity.java

```diff
@@ -99,8 +101,8 @@ protected AbstractEventEntity(final String type, final String instanceId, final
     this.id = id;
     this.title = title;
     this.description = description;
-    this.start = (start != null ? start.toShortISO8601() : null);
-    this.end = (end != null ? end.toShortISO8601() : null);
+    this.start = (start != null ? asIso8601(start, false) : null);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "start" is marked "javax.validation.constraints.NotNull" but is set to null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2637)